src/form-layout.scss

```diff
@@ -0,0 +1,139 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+/*!
+.fd-form-horizontal
+*/
+
+$block: #{$fd-namespace}-form-layout;
+
+.#{$block} {
+  @include fd-reset();
+
+  background-color: transparent;
+  background-color: var(--sapGroup_ContentBackground);
+  padding: 0.5rem 1rem 1rem;
+  grid-gap: var(0.5rem, 0.5rem);
+  width: 100%;
+
+  &_horizontal--4-7-1 {
```

Comment:
I think we can remove the 4-7-1 from the class name.
could be just `fd-form-layout--horizontal` which will be 4-7-1 layout, and `fd-form-layout--vertical` which will be 12

@droshev , what do you think?